monkestation/code/game/machinery/bomb_actualizer.dm

```diff
@@ -0,0 +1,235 @@
+	. = ..()
+	RegisterSignal(src, COMSIG_ATOM_INTERNAL_EXPLOSION, PROC_REF(modify_explosion))
+	radio = new(src)
+	radio.keyslot = new radio_key
+	radio.set_listening(FALSE)
+	radio.recalculateChannels()
+	countdown = new(src)
+	update_appearance()
+
+	//Stolen from anomaly_refinery
+/obj/machinery/bomb_actualizer/attackby(obj/item/tool, mob/living/user, params)
+	if(active && istype(tool, /obj/item/transfer_valve))
+		to_chat(user, span_warning("You can't insert [tool] into [src] while [p_theyre()] currently active."))
+		return
+	if(istype(tool, /obj/item/transfer_valve))
+		if(inserted_bomb)
+			to_chat(user, span_warning("There is already a bomb in [src]."))
+			return
+		var/obj/item/transfer_valve/valve = tool
+		if(!valve.ready())
+			to_chat(user, span_warning("[valve] is incomplete."))
+			return
+		if(!user.transferItemToLoc(tool, src))
+			to_chat(user, span_warning("[tool] is stuck to your hand."))
+			return
+		inserted_bomb = tool
+		tank_to_target = inserted_bomb.tank_two
+		to_chat(user, span_notice("You insert [tool] into [src]"))
+		return
+	update_appearance()
+	return ..()
+
+/obj/machinery/bomb_actualizer/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if(!active)
+		default_unfasten_wrench(user, tool)
+		return TOOL_ACT_TOOLTYPE_SUCCESS
+	return FALSE
+
+/obj/machinery/bomb_actualizer/screwdriver_act(mob/living/user, obj/item/tool)
+	if(!active)
+		if(!default_deconstruction_screwdriver(user, "[base_icon_state]-off", "[base_icon_state]", tool))
+			return FALSE
+	update_appearance()
+	return TRUE
+
+/obj/machinery/bomb_actualizer/crowbar_act(mob/living/user, obj/item/tool)
+	if(!default_deconstruction_crowbar(tool) && !active)
+		return FALSE
+	return TRUE
+
+/**
+ * Starts the Detonation Sequence
+ */
+/obj/machinery/bomb_actualizer/proc/start_detonation()
+	if(active)
+		say("ERROR: The countdown has aready begun!!!")
+		return
+
+	if(!istype(inserted_bomb))
+		say("ERROR: No Bomb Inserted")
+		return
+
+	alerthere = get_area(src)
+	if(!on_reebe(src))
+		say("Beginning detonation sequence. Countdown starting.")
+		countdown.start()
+		active = TRUE
+		next_beep = world.time + 10
+		timer = world.time + (default_time * 10)
+
+		begin_processing()
+		priority_announce("DANGER - Tampering of bluespace ordinance dampeners detected, Resulting explosion may be catastrophic to station integrity \
+					Remove the tampering device within 7 Minutes or evacuate the localized areas. \
+					Location: [alerthere].", "Doppler Array Detection - DANGER", 'sound/misc/notice3.ogg')
+		return
+	say("UNKNOWN ERROR: Nice try nerd. ")
+	return
+
+//Process for handling the bombs timer
+/obj/machinery/bomb_actualizer/process()
+	var/volume = 40
+	if(!active)
+		end_processing()
+		timer = null
+		next_beep = null
+		countdown.stop()
+		stage = 0
+		return
+	if(!isnull(next_beep) && (next_beep <= world.time))
+
+		playsound(loc, beepsound, volume, FALSE)
+		next_beep = world.time +10
+	if(seconds_remaining() == 60)
```

Comment:
are you able to set the timer to an odd number? if so, this won't go off ever

Reply:
No function to change the time added so no